src/components/Error/Error.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { VFC, memo } from 'react';
+import { Box } from '@chakra-ui/react';
+
+interface Props {
+
+  /** Error object. */
+  error: Error;
+}
+
+/**
+ * Error component.
+ */
+const ErrorComponent: VFC<Props> = ({ error }) => (
+  <Box h="full" w="full" display="flex" alignItems="center" justifyContent="center">
```

Comment:
I can't see tags loader because its too far down, because its height calculated strangely (not sure, but it's close to combined height of siblings). I suggest remove it because `height: 100%` introduces many bugs.

```suggestion
  <Box paddingY={4} w="full" display="flex" alignItems="center" justifyContent="center">
```